gpiozero/pins/mock.py

```diff
@@ -413,3 +385,34 @@ def tx_word(self, value, bits_per_word=None):
             bits = reversed(bits)
         self.tx_buf.extend(bits)
 
+
+class MockFactory(LocalPiFactory):
+    def __init__(self, revision='a21041', pin_class=MockPin):
```

Comment:
> Whether pin_class should be ... erm ... not sure.

I was thinking specifically of the other Mock... pins (MockSPIClockPin, MockChargingPin, etc.) but perhaps they're a bit too special-case to be selectable from the command-line, and should always just be selected in code instead.